src/main/java/uk/gov/laa/ccms/caab/service/LookupService.java

```diff
@@ -236,6 +244,112 @@ public Mono<CommonLookupDetail> getApplicationStatuses() {
     return ebsApiClient.getCommonValues(COMMON_VALUE_APPLICATION_STATUS);
   }
 
+  /**
+   * Get a list of matter types for proceedings.
+   *
+   * @return MatterTypeLookupDetail containing the matterType values.
+   */
+  public Mono<MatterTypeLookupDetail> getMatterTypes(final String categoryOfLaw) {
+    return ebsApiClient.getMatterTypes(categoryOfLaw);
+  }
+
+  /**
+   * Get a list of Matter Type Common Values.
+   *
+   * @return CommonLookupDetail containing the common lookup values.
+   */
+  public Mono<CommonLookupDetail> getMatterTypes() {
+    return ebsApiClient.getCommonValues(COMMON_VALUE_MATTER_TYPES);
+  }
+
+  /**
+   * Get a list of proceeding types for proceedings.
+   *
+   * @return CommonLookupDetail containing the common lookup values.
+   */
+  public Mono<ProceedingDetails> getProceedings(
+      final ApplicationDetail application,
```

Comment:
Could this be changed to just take in the flags and ids that it needs? 
It probably doesn't matter, it's just that so far the LookupService hasn't dealt in Applications/Cases and is kept as standalone as possible.

Reply:
amended